### DISCUSSION TITLE
Contest QSOs Missing Callbook Info in Logbook

### DISCUSSION THREAD

**Discussion by msmcguire** [magicbug/Cloudlog#2014] [Ideas / Feature Requests]
When using the contest logging feature, the logged QSOs do not pull in data from QRZ or HamQTH as we see when using the main logbook. Is this by design? The result is that any QSO logged with the contest log page lacks all information except for call and exchange. Is there a way to populate this information for the logbook? Thanks.

**Comment by AndreasK79** [comment]
The contesting logging is a barebone logging feature, so that feature was never implemented.

You can use the advanced logbook and try to update those QSOs there.
